packages/f-searchbox/src/components/formElements/FormSearchField.vue

```diff
@@ -82,7 +84,9 @@ export default {
 
     computed: {
         ...mapState('searchbox', [
-            'shouldInputFieldHaveFocus'
+            'shouldInputFieldHaveFocus',
+            'streetNumberRequired',
```

Comment:
`isStreetNumberRequired`?

Reply:
Mind if i add this to the next PR? Have already updated in the next one 👍

Reply:
Yep – that's cool